tests/integration/test_storage_kerberized_kafka/Dockerfile_kdc

```diff
@@ -0,0 +1,13 @@
+FROM centos:6.6
+# old OS to make is faster and smaller
```

Comment:
@qoega shouldn't we adopt it / put in yandex dockerhub together with other test images? 

@ilejn please put here the image name created by that image (it allows to grep for a dockerfile by looking for an image name), for example like that
https://github.com/ClickHouse/ClickHouse/blob/9e3797b2521ff4352d0c036bf71a077bae21f269/docker/test/stateless/Dockerfile#L1

@qoega - maybe we can use that practice for all Dockerfiles in our repo (i always waste minutes trying to understand which Dockerfile corresponds to which image on dockerhub)

Reply:
We don't have all Dockerfiles that used in tests in CH repo, so it is reasonable to simply remove the Dockerfile, is not it?

Reply:
From my perpective if the contained is not commonly used (like ubuntu/nginx/kafka/etc) and was created especially for clickhouse testing needs, it's better to keep its sources in the repo, and build it together with other containers. 

But i prefer to leave that decision to  @qoega

Reply:
We don't have keys to arenadata DockerHub and thus have no way to rebuild this Docker in CI. Thus we really need to setup this docker build in common way.

It should be trivial:
1) Add new path and name here.  https://github.com/ClickHouse/ClickHouse/blob/master/docker/images.json
2) Correct version that is build in PR can be set as in clients docker compose files
https://github.com/ClickHouse/ClickHouse/commit/e6420c82ca0845bc0987027cf7329c05c937eb70 
3) In the PR above you can also see how it will be passed via runner
4) Currently there is one place in CI configuration that is not available in this repository where we need to add your docker image name. We can make it in advance at any time and your changes will be tested correctly in this PR

Reply:
I've 
- added images.json entry 
- moved Dockerfile_kdc to docker/test/integration/kerberos_kdc
- left tag 'latest'